vdslib/src/vespa/vdslib/distribution/distribution.cpp

```diff
@@ -335,8 +332,8 @@ bool
 Distribution::allDistributorsDown(const Group& g, const ClusterState& cs)
 {
     if (g.isLeafGroup()) {
-        for (uint32_t i=0, n=g.getNodes().size(); i<n; ++i) {
-            const NodeState& ns(cs.getNodeState(Node(NodeType::DISTRIBUTOR, g.getNodes()[i])));
+        for (unsigned short node : g.getNodes()) {
```

Comment:
Please use explicit `uint16_t` for types referencing distribution keys

Reply:
Fixed